File: Natom.Petshop.Gestion.Backend/Natom.Petshop.Gestion.Entities/DTO/FeatureFlagsDTO.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Natom.Petshop.Gestion.Entities.DTO
{
    public class FeatureFlagsDTO
    {
        [JsonProperty("stock")]
        public FeatureFlagsStockDTO Stock { get; set; }
    }

    public class FeatureFlagsStockDTO
    {
        [JsonProperty("permitir_venta_con_stock_negativo")]
        public bool PermitirVentaConStockNegativo { get; set; }
    }
}

File: Natom.Petshop.Gestion.Frontend/src/app/classes/dto/_feature_flags/feature-flags.dto.ts
import { FeatureFlagsAccesoDTO } from "./feature-flags.acceso.dto";
import { FeatureFlagsClientesDTO } from "./feature-flags.clientes.dto";
import { FeatureFlagsStockDTO } from "./feature-flags.stock.dto";

export class FeatureFlagsDTO {
    public acceso: FeatureFlagsAccesoDTO;
    public clientes: FeatureFlagsClientesDTO;
    public stock: FeatureFlagsStockDTO;
}